src/solver/conv_winoRxS.cpp

```diff
@@ -697,6 +705,19 @@ static bool IsApplicableBase(const ExecutionContext& ctx, const ProblemDescripti
     }
 #endif
 
+#if WORKAROUND_ISSUE_2492_TINY_TENSOR
+    if(!miopen::IsDisabled(ENV(MIOPEN_DEBUG_WORKAROUND_ISSUE_2492)) &&
+       !miopen::debug::IsWarmupOngoing)
+    {
+        // Group count is not taken into account intentionally.
+        if(problem.GetInHeight() <= 6     //
+           && problem.GetInHeight() <= 6  //
```

Comment:
```suggestion
           && problem.GetInWidth() <= 6  //
```

Reply:
@junliume Ahaha, leaked from the original #2693. Thanks for catching this! ;)

Reply:
@junliume but formatting! ))) PS I see the fix in 959a7aa0803e64751f36.